types/jsreport-core/index.d.ts

```diff
@@ -0,0 +1,98 @@
+// Type definitions for jsreport-core 1.5
+// Project: http://jsreport.net
+// Definitions by: taoqf <https://github.com/taoqf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// TypeScript Version: 2.6
```

Comment:
This would have to go on line 5 to take effect -- but apparently it isn't needed, so just remove.